logdna/index.js

```diff
@@ -1,32 +1,19 @@
+// Internal Libraries
+const constants = require('./lib/constants.js');
```

Comment:
Let's not separate into 2 or more libraries - for the sake of keeping all lambdas compact and centralized in the future, let's have one `CloudWatch` class defined inside `./lib/cloudwatch.js` where you can set `constants` inside the `constructor` and `createLogs` as a method.

Reply:
In this case, it probably doesn't make sense to encapsulate it into a class. This is only a handler and doesn't need to keep its state. I'd prefer to keep it a function as is.

Reply:
here, that's agreed, by design, @weizou19 - by design, what we want to accomplish is, for lambda to get the logs from each `AWS` service, to have one shared unique `index.js` handler and separate service classes like `cloudwatch.js`, `s3.js`, `sns.js`, `sqs.js`, etc.

We can discuss this more, if you want

Reply:
Here is the question:  what effect does the .zip file's size has besides being un-editable from aws editor? If we add s3 and other lib here, the size of the .zip will grow.